packages/patrol_cli/lib/src/commands/doctor.dart

```diff
@@ -34,35 +35,59 @@ class DoctorCommand extends PatrolCommand {
     return 0;
   }
 
+  void _printHeader() {
+    _logger.info('Patrol doctor:');
+  }
+
   void _printVersion() {
     _logger.info('Patrol CLI version: $version');
   }
 
   void _printAndroidSpecifics() {
-    _checkIfToolInstalled('adb');
+    _logger.info('Android: ');
+
+    _checkIfToolInstalled(
+      'adb',
+      'See the link: https://www.xda-developers.com/install-adb-windows-macos-linux/#how-to-set-up-adb',
+    );
 
     final androidHome = _platform.environment['ANDROID_HOME'];
     if (androidHome?.isNotEmpty ?? false) {
-      _logger.success('Env var \$ANDROID_HOME set to $androidHome');
+      _logger.success('• Env var \$ANDROID_HOME set to $androidHome');
     } else {
-      _logger.err(r'Env var $ANDROID_HOME is not set');
+      final linkHint = switch (_platform.operatingSystem) {
+        Platform.linux => 'https://stackoverflow.com/a/28889144/9899010',
+        Platform.macOS => 'https://stackoverflow.com/a/33631853/9899010',
+        Platform.windows =>
+          'https://www.ibm.com/docs/en/rtw/10.5.1?topic=prwut-setting-changing-android-home-path-in-windows-operating-systems',
```

Comment:
This website shows:

![Screenshot 2023-12-11 at 10 51 51 AM](https://github.com/leancodepl/patrol/assets/40357511/96f86ced-0b4b-4397-8cee-ff51d441eb94)

just change the link to the latest